Tests/XCUITests/NavigationTest.swift

```diff
@@ -194,83 +195,95 @@ class NavigationTest: BaseTestCase {
-        // Tapping two times when the text is not selected will reveal the menu
-        app.textFields["address"].tap()
-        waitForExistence(app.textFields["address"])
-        app.textFields["address"].tap()
-        waitForExistence(app.menuItems["Select All"])
-        XCTAssertTrue(app.menuItems["Select All"].exists)
-        XCTAssertTrue(app.menuItems["Select"].exists)
-
-        // Tap on Select All option and make sure Copy, Cut, Paste, and Look Up are shown
-        app.menuItems["Select All"].tap()
-        waitForExistence(app.menuItems["Copy"])
-        if iPad() {
-            XCTAssertTrue(app.menuItems["Copy"].exists)
-            XCTAssertTrue(app.menuItems["Cut"].exists)
-            XCTAssertTrue(app.menuItems["Look Up"].exists)
-            XCTAssertTrue(app.menuItems["Share…"].exists)
+    func testCopyLinkPrivateMode() throws {
+        if processIsTranslatedStr() == m1Rosetta {
+            throw XCTSkip("Copy & paste may not work on M1")
         } else {
-            XCTAssertTrue(app.menuItems["Copy"].exists)
-            XCTAssertTrue(app.menuItems["Cut"].exists)
-            XCTAssertTrue(app.menuItems["Look Up"].exists)
+            navigator.performAction(Action.CloseURLBarOpen)
+            navigator.nowAt(NewTabScreen)
+            navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+            longPressLinkOptions(optionSelected: "Copy Link")
+            navigator.goto(NewTabScreen)
+            waitForExistence(app.textFields["url"])
+            app.textFields["url"].press(forDuration: 2)
+
+            app.tables.otherElements[ImageIdentifiers.paste].tap()
+            app.buttons["Go"].tap()
+            waitUntilPageLoad()
+            waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
         }
+    }
 
-        app.textFields["address"].typeText("\n")
-        waitUntilPageLoad()
-        waitForNoExistence(app.staticTexts["XCUITests-Runner pasted from Fennec"])
-
-        app.textFields["url"].press(forDuration: 3)
-        app.tables.otherElements[ImageIdentifiers.copyLink].tap()
-
-        sleep(2)
-        app.textFields["url"].tap()
-        // Since the textField value appears all selected first time is clicked
-        // this workaround is necessary
-        waitForNoExistence(app.staticTexts["XCUITests-Runner pasted from Fennec"])
-        app.textFields["address"].tap()
-        waitForExistence(app.menuItems["Copy"])
-        if iPad() {
-            XCTAssertTrue(app.menuItems["Copy"].exists)
-            XCTAssertTrue(app.menuItems["Cut"].exists)
-            XCTAssertTrue(app.menuItems["Look Up"].exists)
-            XCTAssertTrue(app.menuItems["Share…"].exists)
-            XCTAssertTrue(app.menuItems["Paste & Go"].exists)
-            XCTAssertTrue(app.menuItems["Paste"].exists)
+    func testLongPressOnAddressBar() throws {
+        if processIsTranslatedStr() == m1Rosetta {
```

Comment:
Should I disable this test for M1 as well?

Reply:
No, I don't think so... this test checks the options in the ulr bar when we perform a long press... that should work on M1 too as the issue affecting it is related to pasting... 
It works for me locally if I remove the "look up" option.. I don't see that one on the simulator

Reply:
![Simulator Screen Shot - iPhone 14 Pro Max (iOS 16 0) - 2022-11-28 at 12 41 02](https://user-images.githubusercontent.com/1740517/204345226-650ae9e5-05e6-447b-b4a9-a113d6bab284.png)
😞 When the test long press on the awesome bar, the pop-up appears. The screenshot is taken on the iPhone 14 Pro Max iOS 16.0 simulator.
`app.textFields["url"].press(forDuration: 3)`

Reply:
Oh I see ... yeah, then let's disable for now adding a comment that we get that security pop up introduced on iOS 16